matcher/service/openreview_interface.py

```diff
@@ -450,8 +450,20 @@ def _get_all_edges(self, edge_invitation_id):
 
         self.logger.debug(f"Get edges for invitation id={edge_invitation_id}")
 
+        def get_paper_edges(paper_id):
+            grouped_edges = self.client.get_grouped_edges(
+                groupby="head",
+                invitation=edge_invitation_id,
+                head=paper_id,
+                select="head,tail,label,weight",
+            )
+            if grouped_edges and 'values' in grouped_edges[0]:
+                return grouped_edges[0]['values']
+            else:
+                return []
```

Comment:
`get_grouped_edges` returns JSON instead of `openreview.api.Edge` (get_edges and get_all_edges return `openreview.api.Edge`) so you might either need to change how the `result` is used or cast these to Edges

Reply:
Oops just saw your changes!